packages/victory-core/src/index.d.ts

```diff
@@ -232,6 +232,8 @@ export class VictoryLabel extends React.Component<VictoryLabelProps, any> {}
 // #region Victory Container
 
 export interface VictoryContainerProps {
+  "aria-describedby"?: string;
+  "aria-labelledby"?: string;
```

Comment:
I left off `aria-label` since `title` effectively fills that role (using a plain string as a label for screen readers).